modules/xfem/include/userobjects/CircleCutUserObject.h

```diff
@@ -0,0 +1,40 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef CIRCLE_CUT_USEROBJECT_H
+#define CIRCLE_CUT_USEROBJECT_H
```

Comment:
The convention in framework is _not_ to have underscores in the symbol apart from the one in `_H`